bin/sync_django_ldap.py

```diff
@@ -78,24 +80,37 @@ def main():
         logging.exception("Oops")
         parent_pid = 1
 
+    # Set user of child processes
```

Comment:
not that i care much, but why do all this? pass the correct user via cmdline and set apache as default? when you configure this script, i guess you know what user this could be?

Reply:
I'm adding `ldap` as another default to make the default behaviour of `sync_django_ldap` more reliable. The less configuration needed the better. Since we sync data from django to LDAP it is very reasonable to expect an `ldap` user to exists in the host system.

Reply:
Stijn is right, when we deploy this we will know which user it has to use. Probing multiple users seems pointless.

I would even not give it a default and error out when not specified (but will break things for everybody else).

Reply:
I would agree if I was adding a hundred lines of code with additional points of failure, but it is not the case. Regardless, I won't start a discussion about it. Removed in https://github.com/hpcugent/vsc-administration/pull/102/commits/91fac2e645a01b250ccf107e4091026aafb5a872